src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/DistributionScheduler.java

```diff
@@ -15,25 +12,22 @@
  */
 @CoverageIgnore
 @Component
-public class DistributionScheduler implements HealthIndicator {
+public class DistributionScheduler {
   private static final Logger log = LoggerFactory.getLogger(DistributionScheduler.class);
 
-  private DistributionInfo distributionInfo = new DistributionInfo();
+  private ActionDistributor actionDistributorImpl;
 
-  @Autowired private ActionDistributor actionDistributorImpl;
-
-  @Override
-  public Health health() {
-    return Health.up().withDetail("distributionInfo", distributionInfo).build();
+  public DistributionScheduler(ActionDistributor actionDistributorImpl) {
+    this.actionDistributorImpl = actionDistributorImpl;
   }
 
   /** Scheduled execution of the Action Distributor */
   @Scheduled(fixedDelayString = "#{appConfig.actionDistribution.delayMilliSeconds}")
   public void run() {
-    try {
-      distributionInfo = actionDistributorImpl.distribute();
-    } catch (final Exception e) {
-      log.error("Exception in action distributor", e);
-    }
+    DistributionInfo distInfo = actionDistributorImpl.distribute();
```

Comment:
this is gonna generate a lot of log if it's running every 1 second. Maybe only log if the count is nonzero?